section2/exercises/if-statements.js

```diff
@@ -82,15 +93,24 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
-var hasSauce = true;
+var cupsOfFlour = 3;
+var hasSauce = false;
 
 // Write your conditional here
+if (cupsOfFlour >= 2 && hasSauce === true){
+  console.log("I can make pizza");
+} else {
+  console.log("I cannot make pizza");
+}
 
 
 /*
```

Comment:
Your style and syntax (use of indentation, white space, spaces) is EXCELLENT and very consistent.